### PR TITLE
update .net standard topic

### LINKED ISSUE #2927: Add info on how to consume .NET Standard 2.0 libraries from VS2015
Need to add info about how to consume .NET Standard 2.0 libraries into the .NET Standard topic. You can do it with VS 2015 too if you have the latest NuGet client.

https://github.com/dotnet/announcements/issues/24

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9eb8b774-c6f0-8235-6964-15219baf2ea7
* Version Independent ID: 6819a4a7-7527-d8c7-e132-767f7344e00e
* Content: [.NET Standard](https://docs.microsoft.com/en-us/dotnet/standard/net-standard#feedback)
* Content Source: [docs/standard/net-standard.md](https://github.com/dotnet/docs/blob/master/docs/standard/net-standard.md)
* Product: **dotnet**
* GitHub Login: @mairaw
* Microsoft Alias: **mairaw**

### PR DESCRIPTION
**Pull request by mairaw**
Fixes #2927 

@terrajobst @davidfowl can you please take a look and see if there's anything else that should be updated?

I've moved all the PCL info to the bottom.

[Internal review URL](https://review.docs.microsoft.com/en-us/dotnet/standard/net-standard?branch=pr-en-us-6556)

### CODE REVIEW

File: docs/standard/net-standard.md [hunk 1]
```diff
@@ -107,6 +93,38 @@ APIs added to any of the implementations (such as, .NET Framework, .NET Core and
 - Use libraries that depend on the same .NET Standard version or lower.
 - If you find a library that depends on a higher .NET Standard version, you either need to adopt that same version or decide not to use that library.
 
+## Targeting .NET Standard
+
+You can [build .NET Standard Libraries](../core/tutorials/libraries.md) using a combination of the `netstandard` framework and the NETStandard.Library metapackage. You can see examples of [targeting the .NET Standard with .NET Core tools](../core/packages.md).
+
+## .NET Framework compatibility mode
+
+Starting with .NET Standard 2.0, the .NET Framework compatibility mode was introduced. This compatibility mode allows .NET Standard projects to reference .NET Framework libraries as if they were compiled for .NET Standard. Referencing .NET Framework libraries doesn't work for all projects, such as if the library uses Windows Presentation Foundation (WPF) APIs.
```
Comment (rpetrusha) [inline docs/standard/net-standard.md]: nit: such as if the library uses --> such as libraries that use

File: docs/standard/net-standard.md [hunk 2]
```diff
@@ -87,11 +73,11 @@ Derivative artifacts are provided to enable more convenient reading and to enabl
...
+The primary distribution vehicle for the .NET Standard reference assemblies is [NuGet packages](../core/packages.md). Implementations are delivered in a variety of ways, appropriate for each .NET implementation.
 
-NuGet packages target one or more [frameworks](frameworks.md). The .NET Standard packages target the ".NET Standard" framework. You can target the .NET Standard Framework using the `netstandard` [compact TFM](frameworks.md) (for example, `netstandard1.4`). Libraries that are intended to run on multiple runtimes should target this framework. 
+NuGet packages target one or more [frameworks](frameworks.md). The .NET Standard packages target the ".NET Standard" framework. You can target the .NET Standard Framework using the `netstandard` [compact TFM](frameworks.md) (for example, `netstandard1.4`). Libraries that are intended to run on multiple runtimes should target this framework. If you want a broader set of APIs, you should target `netstandard2.0` since the number of available APIs more than doubled between .NET Standard 1.6 and 2.0.
```
Comment (rpetrusha) [inline docs/standard/net-standard.md]: - nit: should framework in .NET Standard Framework be capitalized (I suspect not)
- The "if you want" sentence seems out of place. I think a slight rewrite helps: "You can target the .NET Standard framework using the \`netstandard\` \[compact target framework moniker](link) (TFM). Libraries that are intended to run on multiple platforms should target at least \`netstandard1.4\`. For the broadest set of APIs, target...

File: docs/standard/net-standard.md [hunk 3]
```diff
@@ -107,6 +93,38 @@ APIs added to any of the implementations (such as, .NET Framework, .NET Core and
...
+
+## .NET Framework compatibility mode
+
+Starting with .NET Standard 2.0, the .NET Framework compatibility mode was introduced. This compatibility mode allows .NET Standard projects to reference .NET Framework libraries as if they were compiled for .NET Standard. Referencing .NET Framework libraries doesn't work for all projects, such as if the library uses Windows Presentation Foundation (WPF) APIs.
+
+For more information, see [.NET Framework compatibility mode](../core/porting/third-party-deps.md#net-framework-compatibility-mode).
+
+## .NET Standard libraries and Visual Studio
+
+In order to build .NET Standard libraries in Visual Studio, make sure you have [Visual Studio 2017 version 15.3](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=button+cta&utm_content=download+vs2017) or later installed on Windows or [Visual Studio for Mac version 7.1](https://visualstudio.microsoft.com/vs/visual-studio-mac/) or later installed on macOS.
```
Comment (rpetrusha) [inline docs/standard/net-standard.md]: comma after Windows

File: docs/standard/net-standard.md [hunk 4]
```diff
@@ -107,6 +93,38 @@ APIs added to any of the implementations (such as, .NET Framework, .NET Core and
...
+
+## Comparison to Portable Class Libraries
+
+.NET Standard is the replacement for [Portable Class Libraries (PCL)](./cross-platform/cross-platform-development-with-the-portable-class-library.md). The .NET Standard improves on the experience of creating portable libraries by curating a standard BCL and establishing greater uniformity across .NET implementations as a result. A library that targets .NET Standard is a PCL or a ".NET Standard-based PCL". Existing PCLs are "profile-based PCLs".
+
+.NET Standard and PCL profiles were created for similar purposes but also differ in key ways.
+
+Similarities:
+
+- Defines APIs that can be used for binary code sharing.
```
Comment (rpetrusha) [inline docs/standard/net-standard.md]: Defines --> Define (they is the implicit subject)

File: docs/standard/net-standard.md [hunk 5]
```diff
@@ -107,6 +93,38 @@ APIs added to any of the implementations (such as, .NET Framework, .NET Core and
...
+
+Differences:
+
+- .NET Standard is a curated set of APIs, while PCL profiles are defined by intersections of existing platforms.
+- .NET Standard linearly versions, while PCL profiles do not.
+- PCL profiles represents Microsoft platforms while the .NET Standard is agnostic to platform.
```
Comment (rpetrusha) [inline docs/standard/net-standard.md]: agnostic to platform --> platform-agnostic

**Review by rpetrusha — APPROVED** [review]
LGTM, @mairaw. I suggested a number of a changes and rewordings, but they're all nits.

**Comment by mairaw** [comment]
Thank you @rpetrusha for your feedback. Comments addressed.

**Review by terrajobst — APPROVED** [review]
LGTM!

**Pull request merged**